src/Features/SupportTesting/MakesAssertions.php

```diff
@@ -164,4 +165,28 @@ public function assertReturned($value)
 
         return $this;
     }
+
+    public function assertViewIs($value)
+    {
+        if (! $this->lastState->getView() instanceof View) {
+            PHPUnit::fail('The response is not a view.');
+        }
+
+        // When the render method exists on the component, the view name will be the actual name...
+        if (method_exists($this->lastState->getComponent(), 'render')) {
+            PHPUnit::assertEquals($value, $this->lastState->getView()->name());
+
+            return $this;
+        }
+
+        // When the render method does not exist on the component, the view name will be the full path...
+        $fileName = str($value)->replace('.', '/')->__toString();
+
+        PHPUnit::assertThat(
+            $this->lastState->getView()->name(),
+            PHPUnit::equalTo(resource_path('views/').$fileName.'.blade.php'),
```

Comment:
I haven't dug too deep here, but is there a better way than hardcoding `resource_path('views/')`? Some apps change their view directory, and there can also be multiple. They can also use a non `.blade.php` suffix (just `.php` in some cases). I know this is edge casey-stuff, but I just want to make sure there's not some kind of Laravel method somewhere or precedent that we can use here.

Reply:
While no `render()` method is given, we assume the view is located in: 
```php
View::file($viewPath . '/' . $fileName . '.blade.php')
```
In that regard, I think it is fine to use `.blade.php` as suffix.

Reply:
As for `resource_path('views/')`, I'm not sure. As long as you customize your view directory to something within `resources/views` it should be fine (as covered within the test). Not sure what happens when you cutomize to something totally different, but that might be a very-very edge case.